distribution/docs/src/main/resources/_contents/_data/media-attributes-table-contents.adoc

```diff
@@ -20,16 +20,16 @@ name for such, note that this is not the mime type
 |The file format version of the resource.
 |String
 |< 1024 characters
-| 
+|Word 2016
```

Comment:
I don't think Tika uses any particular formatting. In the case of word docs, Microsoft's convention seems to be `Word <date-range> Document (*.<file extension>)`. So I guess `format` would be `docx` and `format-version` would be `2016`? For other extensions it could be weirder, e.g. `doc` and `1997-2003`.